app/controllers/Api/V1/playlists_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Api::V1::PlaylistsController < ApplicationController
+  def songs
+    token = params[:token]
+    # set defual songs & tempo
+    num_songs = 10
```

Comment:
not sure we need to define `num_songs` and `tempo` here, unless it is for a default